sklearn/linear_model/ridge.py

```diff
@@ -281,6 +286,12 @@ def ridge_regression(X, y, alpha, sample_weight=None, solver='auto',
     -----
     This function won't compute the intercept.
     """
+    if return_intercept:
```

Comment:
in the fit(), [here](https://github.com/scikit-learn/scikit-learn/pull/5360/files#diff-489e81d5edaa5bb1d352988699773551R454)